### ISSUE TITLE
Enchancement - Cancel download

### ISSUE THREAD

**Issue by scrubbless** [NPBruce/valkyrie#1122]
### Description of Problem or Question
This is not a huge issue, but if for any reason a scenario hangs when downloading or takes a long time, there is no way to cancel the download.

A button would be helpful, something like this would be fine
![image](https://user-images.githubusercontent.com/32058089/54743780-0cf81d80-4bbd-11e9-9ee4-8aa1901209cb.png)

### Valkyrie Version
2.2.004

**Comment by chrismaliszewski** [comment]
Good idea. Personally, I'd prefer the progress bar and Cancel buttons the order way around:
![image](https://user-images.githubusercontent.com/7532602/54785999-f804a400-4c27-11e9-98ac-fd3e935583dd.png)

**Comment by redwolf2** [comment]
Good idea, but I think the text "Cancel" should be enough.

**Comment by scrubbless** [comment]
Yeh, agreed.
